src/EditorFeatures/Core/FindUsages/IFindUsagesService.cs

```diff
@@ -2,59 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.FindUsages;
 using Microsoft.CodeAnalysis.Host;
 
 namespace Microsoft.CodeAnalysis.Editor.FindUsages
 {
-    [Obsolete("Legacy API for TypeScript.  Once TypeScript moves to IVSTypeScriptFindUsagesService", error: false)]
```

Comment:
typescript has moved to IVSTypeScriptFindUsagesService, so we do not need the bridge interface.